python/pylibraft/pylibraft/distance/CMakeLists.txt

```diff
@@ -22,8 +22,5 @@ rapids_cython_create_modules(
     CXX
     SOURCE_FILES "${cython_sources}"
     LINKED_LIBRARIES "${linked_libraries}"
+    ASSOCIATED_TARGETS cuml
```

Comment:
Is this a copy/paste error?

Reply:
And here I was thinking I was careful... thanks :)